tests/baselines/reference/controlFlowForCatchAndFinally.types

```diff
@@ -120,7 +120,7 @@ class Foo {
 >Aborter : typeof Aborter
 
         } catch (error) {
->error : any
+>error : unknown

```

Comment:
double-checking -- does this test, and the others whose baselines changed, have strict on?

Reply:
Yes.

Reply:
(We should probably dump the flags at the top of test baselines)